addons/website/static/src/scss/website.wysiwyg.scss

```diff
@@ -63,6 +63,26 @@
     .o_color_combinations_edition {
         margin-top: $o-we-sidebar-content-field-spacing;
     }
+
+    // Snippet filter input
+    .o_snipet_search_filter_input {
+        padding: 7px 9px;
+        transition: all 0.3s ease 0s;
+        transition-property: border;
+        margin-top: $o-we-sidebar-content-field-spacing;
+        width: 100%;
+        border: $o-we-sidebar-content-field-border-width solid $o-we-sidebar-content-field-border-color;
+        background-color: $o-we-sidebar-content-field-input-bg;
+        color: inherit;
```

Comment:
Nice that you thought about using the scss variables :) But use it for the padding rules too. There are we-input in the left panel, that input should have the same paddings as well.

Also try to reorder your scss properties https://github.com/odoo/odoo/wiki/SCSS-coding-guidelines